app/src/main/java/org/mozilla/fenix/StrictModeManager.kt

```diff
@@ -49,9 +50,11 @@ class StrictModeManager(
      * - a lint check: to ensure this value gets used instead of functions that work around it
      * - code owners: to prevent modifications to these above items without perf knowing
      * to make suppressions a more deliberate act.
+     *
+     * This is an Atomic* so it can be incremented from any thread.
```

Comment:
nit: *safely* incremented ... the non-atomic one we can increment from any thread as well :)